src/SuperstateToken.sol

```diff
@@ -89,11 +91,14 @@ contract SuperstateToken is ISuperstateToken, ERC20Upgradeable, PausableUpgradea
     /// @notice Address of the AllowList contract which determines permissions for transfers
     IAllowListV2 public allowListV2;
 
+    /// @notice The address of the contract used to facilitate atomic redemptions, if such a contract exists.
```

Comment:
this should be `protocol redemptions`

Reply:
If/when we use `RedemptionYield`, does this preclude us from using both at the same time?